Dockerfile

```diff
@@ -1,48 +1,50 @@
-FROM quay.io/keycloak/keycloak:19.0.3-legacy
+FROM quay.io/keycloak/keycloak:19.0.3 as builder
 
 LABEL maintainer="support@integrationeye.com"
 LABEL builder="Integsoft s.r.o"
 LABEL description="MBTA Keycloak"
 
 ENV INSTALL_FOLDER=/tmp/files
+ENV KC_FOLDER=/opt/keycloak
+
+ENV KC_DB=mariadb
+ENV KC_HTTP_RELATIVE_PATH=/auth
+ENV KC_CACHE_CONFIG_FILE=cache-ispn-jdbc-ping.xml
+ENV KC_HOSTNAME_STRICT=false
+ENV KC_HTTP_ENABLED=true
+ENV KC_LOG_LEVEL=INFO,cz.integsoft:debug
+ENV KC_PROXY=edge
 
 USER root
 # install system tools and update system
 RUN ["/bin/bash", "-c", "microdnf update -y && microdnf install -y vim"]
-# remove root password
-# RUN ["/bin/bash", "-c", "passwd --delete root"]
 
-USER jboss
+USER keycloak
 
 # copy build scripts and related data
 COPY files/ $INSTALL_FOLDER/
 
-# copy CLI scripts (in this version it is not supported to use startup-scripts variable
-COPY files/startup-scripts/* /opt/jboss/startup-scripts/
+# copy the custom cache config file into the keycloak conf dir
+RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/conf/cache-ispn-jdbc-ping.xml $KC_FOLDER/conf/"]
+
 # copy custom modules
-RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/modules/* $JBOSS_HOME/modules/"]
+RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/modules/* $KC_FOLDER/providers/"]
 
 # copy welcome page
-RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/welcome-content/* $JBOSS_HOME/welcome-content/"]
+#RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/welcome-content/* $KC_FOLDER/welcome-content/"]
```

Comment:
Question: Is this line commented out intentionally?

Reply:
Yes, it is not needed, I removed it.